.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,9 @@
-<!--
-Note: This checklist is a reminder of our shared engineering expectations. Feel free to change it, though reviewer and all items in bold are required.
--->
+Asana Issue #: 
 
-Asana:
+## Description
 
-**Description**:
 
 
-**Steps to test this PR**:
-1. <STEP 1>
-
-
-###### Author Checklist:
-- [ ] Get advice or leverage existing code
-- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
-- [ ] Ensure that there is a testing strategy (and document any non-automated tests)
-- [ ] Consider implications (CPU, memory, database / preference migrations, statistics)
-
-
-###### Reviewer Checklist:
-- [ ] **Ensure the PR solves the problem**
-- [ ] **Review every line of code**
-- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
-- [ ] **Get help if you're uncomfortable with any of the above!**
-- [ ] Determine if there are any quick wins that improve the implementation
+## Steps to Test this PR:
+1. 
+1. 
```

Comment:
2?

Reply:
Nah, markdown is happy with repeatedly using `1.`. it automatically turns it into incrementing numbers and saves you having to manually number them.